### ISSUE TITLE
Running minimesos up twice should log that cluster is already up

### ISSUE THREAD

**Issue by frankscholten** [ContainerSolutions/minimesos#230]
Currently it tried to pull images again

```
frank@franktop:~$ minimesos
export MINIMESOS_MASTER=http://172.17.0.4:5050
export MINIMESOS_MARATHON=http://172.17.0.5:8080
frank@franktop:~$ minimesos up
Pulling images required to launch minimesos ...
Finished pulling images.
```

**Comment by sadovnikov** [comment]
pulling was not actually happening, but I took these log statements from bash script out.

```
vikmac:mini-mesos vik$ ./bin/minimesos up
You are running minimesos on OS X, enabling --exposedHostPorts
export MINIMESOS_ZOOKEEPER=zk://192.168.99.100:2181
export MINIMESOS_MASTER=http://192.168.99.100:5050
export MINIMESOS_MARATHON=http://192.168.99.100:8080

vikmac:mini-mesos vik$ ./bin/minimesos up
You are running minimesos on OS X, enabling --exposedHostPorts
Cluster 3937002904 is already running

vikmac:mini-mesos vik$ ./bin/minimesos info
Minimesos cluster is running: 3937002904
Mesos version: 0.25.0
export MINIMESOS_ZOOKEEPER=zk://172.17.0.3:2181
export MINIMESOS_MASTER=http://172.17.0.4:5050
export MINIMESOS_MARATHON=http://172.17.0.5:8080
```

There are still problems:
- [ ] deserialized cluster does not has its `exposedHostPorts` set to correct value

**Comment by sadovnikov** [comment]
done

```
vikmac:mini-mesos vik$ ./bin/minimesos up
You are running minimesos on OS X, enabling --exposedHostPorts
export MINIMESOS_ZOOKEEPER=zk://192.168.99.100:2181
export MINIMESOS_MASTER=http://192.168.99.100:5050
export MINIMESOS_MARATHON=http://192.168.99.100:8080
vikmac:mini-mesos vik$ 
vikmac:mini-mesos vik$ ./bin/minimesos up
You are running minimesos on OS X, enabling --exposedHostPorts
Cluster 4268631348 is already running
vikmac:mini-mesos vik$ 
vikmac:mini-mesos vik$ ./bin/minimesos info
Minimesos cluster is running: 4268631348
Mesos version: 0.25.0
export MINIMESOS_ZOOKEEPER=zk://192.168.99.100:2181
export MINIMESOS_MASTER=http://192.168.99.100:5050
export MINIMESOS_MARATHON=http://192.168.99.100:8080
```

**Comment by sadovnikov** [comment]
resolved by #233
